locations/spiders/east_of_england_coop_gb.py

```
@@ -45,6 +45,14 @@ def parse(self, response: Response, **kwargs: Any) -> Any:
             item.update(self.EAST_OF_ENGLAND_COOP)
             apply_category(Categories.SHOP_FUNERAL_DIRECTORS, item)
 
+        try:
+            item["opening_hours"] = self.parse_opening_hours(response)
+        except Exception:
+            pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)